examples/multidimensional_array.html

```diff
@@ -0,0 +1,176 @@
+                Array.from(Array(15).keys()).map((value) => (value + 1) ** 5),
+                Array.from(Array(15).keys()).map((value) => (value + 1) ** 6),
+                Array.from(Array(15).keys()).map((value) => (value + 1) ** 7),
+                Array.from(Array(15).keys()).map((value) => (value + 1) ** 8),
+    ],
+            [
+                [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14],
+                ["A4", "B4", "C4", "D4", "E4", "F4", "G4", "H4", "I4", "J4", "K4", "L4", "M4", "N4", "O4"],
+                Array.from(Array(15).keys()).map((value) => value % 2 === 0),
+                Array.from(Array(15).keys()).map((value) => (value + 1) * 2),
+                Array.from(Array(15).keys()).map((value) => (value + 1) * 3),
+                Array.from(Array(15).keys()).map((value) => (value + 1) ** 2),
+                Array.from(Array(15).keys()).map((value) => (value + 1) ** 3),
+                Array.from(Array(15).keys()).map((value) => (value + 1) ** 4),
+                Array.from(Array(15).keys()).map((value) => (value + 1) ** 5),
+                Array.from(Array(15).keys()).map((value) => (value + 1) ** 6),
+                Array.from(Array(15).keys()).map((value) => (value + 1) ** 7),
+                Array.from(Array(15).keys()).map((value) => (value + 1) ** 8),
+    ],
+            [
+                [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14],
+                ["A5", "B5", "C5", "D5", "E5", "F5", "G5", "H5", "I5", "J5", "K5", "L5", "M5", "N5", "O5"],
+                Array.from(Array(15).keys()).map((value) => value % 2 === 0),
+                Array.from(Array(15).keys()).map((value) => (value + 1) * 2),
+                Array.from(Array(15).keys()).map((value) => (value + 1) * 3),
+                Array.from(Array(15).keys()).map((value) => (value + 1) ** 2),
+                Array.from(Array(15).keys()).map((value) => (value + 1) ** 3),
+                Array.from(Array(15).keys()).map((value) => (value + 1) ** 4),
+                Array.from(Array(15).keys()).map((value) => (value + 1) ** 5),
+                Array.from(Array(15).keys()).map((value) => (value + 1) ** 6),
+                Array.from(Array(15).keys()).map((value) => (value + 1) ** 7),
+                Array.from(Array(15).keys()).map((value) => (value + 1) ** 8),
+    ],
+];
+
+// Global variables declared, to be used later
+
+var num_table; // Current table-dimension number
+const tables = document.getElementsByTagName("tables")[0]; // Select the tag tables
+
+function dataListener(x0, y0, x1, y1) {
+            return {
+                num_rows: DATA[num_table][0].length,
+                num_columns: DATA[num_table].length,
+                data: DATA[num_table].slice(x0, x1).map((col) => col.slice(y0, y1)),
+    };
+}
+
+// The createTables() function takes the 3D array data
+// and displays individual 2D array of individual dimensions
+
+function createTables() {
```

Comment:
I think using regular-table to present high dimensional data in an interactive way is neat, and I really like the idea. However, your `createTables` function here is not the best way to go about it.

Even though your underlying data model is 3D, the regular-tables you create here in `createTables` are just 2D. Especially since there isn't any relationship between the slices of `DATA`, just stacking a bunch of tables in a column doesn't make for the most interesting example here.

I think we can do better.